### PR TITLE
HARMONY-451: Fix fair queueing queries for postgres

### PR DESCRIPTION
**Pull request by indiejames**
Fixes an issue with the queries not working on postgres

### CODE REVIEW

File: tasks/service-runner/config/service-template.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     name: $SERVICE_NAME
 spec:
-  replicas: 1
+  replicas: 5
```
Comment (chris-durbin) [inline tasks/service-runner/config/service-template.yaml:8]: revert this change

**Review by chris-durbin — APPROVED** [review]
Paired with James while testing this against postgres - looks good.

File: tasks/service-runner/config/service-template.yaml (hunk 1 above)
Comment (indiejames) [reply tasks/service-runner/config/service-template.yaml:8]: done

**Review by bilts — APPROVED** [review]
